packages/quantic/force-app/main/default/lwc/quanticResult/quanticResult.js

```diff
@@ -47,16 +51,98 @@ export default class QuanticResult extends LightningElement {
     }
 
     const lower = fileType.toLowerCase();
-    if (lower.indexOf('youtube') !== -1) {
-      return 'doctype:video';
+    switch (lower) {
+      case 'youtubevideo':
+        return 'custom:custom99';
+      case 'kb_knowledge':
+        return 'standard:knowledge';
+      case 'html':
+        return 'standard:document';
+      case 'doc':
+        return 'doctype:gdoc';
+      case 'xls':
+        return 'doctype:excel';
+      default:
+        return `doctype:${lower}`;
+    }
+  }
+
+  get label() {
+    if (this.objectTypeLabel) {
+      return this.objectTypeLabel;
+    }
+    if (this.fileTypeLabel) {
+      return this.fileTypeLabel;
+    }
+    if (this.sourceTypeLabel) {
+      return this.sourceTypeLabel;
+    }
+    
+    
+    return `Documentation`
+  }
+
+  get objectTypeLabel() {
+    const objType = this.result.raw.objecttype;
+    if (!objType) {
+      return undefined;
+    }
+
+    const lower = objType.toLowerCase();
+    switch (lower) {
+      case 'feeditem':
```

Comment:
Instead of matching the labels by hand, I would suggest to add localized string and name them so we can dynamically build the label key using `objType`.